app/views/users/show.html.erb

```diff
@@ -1,9 +1,18 @@
-<!DOCTYPE html>
-<html>
-  <head>
-    <meta http-equiv="Content-Type" content="text/html; charset=utf-8">
-  </head>
-    <h1>Details of particular user</h1>
-  <body>
-  </body>
-</html>
\ No newline at end of file
+<!-- locals is a way to pass local variables to a partial -->
+<%= render partial: 'users/user', locals: { user: @user } %>
+<div>
+  <h3>Bio</h3>
+  <p><%= @user.bio %></p>
+</div>
+<hr><hr>
+
+<section id="posts-container">
+  <% @user.posts.limit(3).each do |post| %>
+    <%= render partial: 'posts/post', locals: { post: post } %>
+  <% end %>
```

Comment:
| @bhushan354 Design | Microverse Design |
|-|-|
| ![image](https://github.com/bhushan354/Blog-App-Rails/assets/93039359/f3163c11-01a3-469b-bbf0-fc0f2f9a08c0) | ![image](https://github.com/bhushan354/Blog-App-Rails/assets/93039359/f864cd30-457a-45fc-ad9c-d6cece5e2d53) |

- [ ] It would be nice if you adjusted the comments and likes counter within the boxes of each post because you can see that the comments and likes counters of first post are overlapping the second post box 😨 Kindly fix this to make your app look professional 👍